geomesa-ui/src/main/ui/src/app/map/map.js

```diff
@@ -42,6 +46,20 @@ angular.module('geomesa.map', [])
                     }
                 };
 
+                scope.map.on('singleclick', function(evt) {
+                    var viewResolution = olView.getResolution();
+                    var url = wmsSource.getGetFeatureInfoUrl(
+                        evt.coordinate, viewResolution, 'EPSG:3857',
+                        {'INFO_FORMAT': 'application/json', FEATURE_COUNT: 50}
+                    );
+                    $http.get(url).success(function(data, status, headers, config) {
```

Comment:
Does it make sense to separate out this logic into a service. There is not much going on here currently, but it seems like some of this could be organized separately.